src/consumer/itinerary-item/chat.ts

```diff
@@ -341,11 +342,22 @@ export class Chat implements IEntity, IDestroyable {
           this.messages = messageArray
           // Get the moderators and add
           this.configuration = resp.payload.configuration
+          this.subscriptionManager.removeSubscription(correlationId)
```

Comment:
I don't think you want to do this? Successfully connects, removes the event listener, reconnects, emits, gets message and the listeners isn't there anymore so nothing will happen?

As an aside I don't think ChatGet should even be using `subscriptionManager` as after the initial`ChatGet` we don't receive updates to this. That's all handled on the receive/remove events above in `setupChatListeners`, that's my fault sorry. Just means we are maintaining an unneeded listener, but will be cleaned up on class destroy so we aren't leaking anything. I'll create an issue to remind myself :)